src/components/ButtonGroup/ButtonGroup.scss

```diff
@@ -14,60 +15,76 @@
     display: flex;
     align-items: center;
     justify-content: center;
-
+    gap: $spacing-small;
     .l3-style-button {
-      border: 1px solid;
-      @include theme-prop(background-color, primary-background-color);
-      @include theme-prop(border-color, ui-border-color);
+      @include theme-prop(background-color, secondary-background-color);
+      border-radius: 60px;
```

Comment:
in a globas-css-settings is variable border-radius-bg or large, please find it and  use 


(don't forget to use global styles ) !